supertylerc/died
# Fix $CHILD_STATUS

Closes #24.

Changes:

+ Add a `fail` unless shell command is successful
+ Change `$CHILD_STATUS.exitstatus` to `$CHILD_STATUS.success?` to
  better detect if a shell command failed

Caveats: None

Test Plan:

* Verify build passes
* Check TravisCI output and make sure there were no errors breaking the
  build
